membership-attribute-service/app/services/AttributesFromZuora.scala

```diff
@@ -84,9 +86,13 @@ class AttributesFromZuora(implicit val executionContext: ExecutionContext, syste
           .map(_.maxBy(_.toDateTimeAtStartOfDay.getMillis))
           .map(date => ZuoraAttributes(identityId, DigitalSubscriptionExpiryDate = Some(date)))
         maybeNonGifteeAttributes <- EitherT(AttributesMaker.zuoraAttributes(identityId, subscriptions, paymentMethodForPaymentMethodId, forDate).map(\/.right[String, Option[ZuoraAttributes]]))
+        supporterProductDataAttributes <- EitherT(supporterProductDataService.getAttributes(identityId))
```

Comment:
I guess dynamo is really quick, but this will happen after zuora has returned its results, so would reduce the overall concurrency.  Could put it in a val before the for comprehension to encourage it to run in parallel.

Reply:
Good suggestion